volumina/imageView2D.py

```diff
@@ -166,6 +171,48 @@ def __init__(self, parent, cropModel, imagescene2d):
         # For screen recording BlankCursor doesn't work
         #self.hiddenCursor = QCursor(Qt.ArrowCursor)
 
+    def _onRotateLeft(self):
+        print "-----------------------------------------------------------------------------------------------------"
```

Comment:
Eliminate all print statements, please.  Feel free to use `logger.debug()` instead.